src/haconiwa/task/manager.py

```
@@ -70,7 +84,7 @@ def create_task(self, config: Dict[str, Any]) -> bool:
             logger.error(f"Failed to create task: {e}")
             return False
     
-    def _create_worktree(self, task_name: str, branch: str, space_ref: str) -> bool:
+    def _create_worktree(self, task_name: str, branch: str, space_ref: str, config: Dict[str, Any]) -> bool:
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)